pkg/cnab/config-adapter/adapter.go

```diff
@@ -224,6 +223,7 @@ func (c *ManifestConverter) buildDefaultPorterParameters() []manifest.ParameterD
 				EnvironmentVariable: "PORTER_DEBUG",
 			},
 			Schema: definition.Schema{
+				ID:          "https://porter.sh/schema/bundle.json#porter-debug",
```

Comment:
No, it isn't meant to be a resolvable URI. Just a way for us to say "porter made this". In the future we could publish something and explain our bundle.json though.

Reply:
The wording in the [docs](https://json-schema.org/understanding-json-schema/structuring.html#id) is a bit ambiguous on this detail (_"This provides a unique identifier for the schema, as well as, in most cases, indicating where it may be downloaded."_).  I know for the CNAB schemas we've gotten in the habit of populating `id` with a resolvable URI.  That being said, I'd be fine revisiting this detail at a later date.